OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -84,7 +84,7 @@ public func fixture(filePath: String, status: Int32 = 200, headers: [NSObject: A
  *            and can be later used to remove it with `removeStub:`
  */
 #if swift(>=3.0)
-public func stub(condition: @escaping OHHTTPStubsTestBlock, response: @escaping OHHTTPStubsResponseBlock) -> OHHTTPStubsDescriptor {
+@discardableResult public func stub(condition: @escaping OHHTTPStubsTestBlock, response: @escaping OHHTTPStubsResponseBlock) -> OHHTTPStubsDescriptor {
```

Comment:
Just nitpicking here, but as a a code style preference, I'd have put the `@discardableResult` attribute on its own line (just before the function declaration line)